renderer/utils/utils.ts

```diff
@@ -100,8 +120,17 @@ export function parseEventTitle(event, availableProjects: Array<string>) {
       break;
 
     case 3:
+      if (allProjects.includes(items[1].toLowerCase())) {
+        const activities = latestProjAndAct[items[1].toLowerCase()];
+
+        if (activities && activities.includes(items[1].toLowerCase())) {
+          event.activity =
+            activities[activities.indexOf(items[1].toLowerCase())];
+        } else event.activity = items[1];
+  
+      } else event.activity = items[1];
+
```

Comment:
only as an idea. We can declare in each case vars like: 
const project = items[0].to lowerCase()
const activity = items[1].to lowerCase()
...
To make code more readable for new users